R/html_dependency.R

```diff
@@ -478,10 +496,37 @@ renderDependencies <- function(dependencies,
 
     # add scripts
     if (length(dep$script) > 0) {
+      if (length(dep$script) == 1) dep$script <- list(src = dep$script[[1]])
```

Comment:
It will probably be best to extract this script handling code into a function (which can be defined within `renderDependencies`, since the input value for `dep$script` could either represent a single object with a structure like `list(src="a.js", integrity="abc")`, or a list of such objects. (And additionally, it must handle values like `list("a.js", "b.js")`.